src/controllers/connectionManager.ts

```diff
@@ -814,6 +819,53 @@ export default class ConnectionManager {
         }
     }
 
+    public async refreshAzureAccountToken(uri: string): Promise<boolean> {
```

Comment:
doesn't seem like the boolean is used anywhere, so the return type could just be `Promise<void>`